benchexec/runexecutor.py

```diff
@@ -745,6 +749,13 @@ def execute_run(
             if files_size_limit < 0:
                 sys.exit(f"Invalid files-size limit {files_size_limit}.")
 
+        self.parent_setup_fn = (
+            util.dummy_fn if parent_setup_fn is None else parent_setup_fn
+        )
+        self.parent_cleanup_fn = (
+            util.dummy_fn if parent_cleanup_fn is None else parent_cleanup_fn
+        )
```

Comment:
Why store this in attributes? AFAIS the only use is in `_execute`, which is directly called from this method.